docs/docs/Components/components-overview.md

```diff
@@ -24,7 +16,7 @@ During the flow creation process, you will notice handles (colored circles) atta
 On the top right corner of the component, you'll find the a play button to run a component. Once it runs, a status icon appears and you can hover over that to visualize success or error messages. Start interacting with your AI by clicking the **Playground** at the bottom right of the workspace.
 
 
-### Component Menu {#7e3f2f8ff5074b2fb3eee97c9cfaabe7}
+## Component Menu {#7e3f2f8ff5074b2fb3eee97c9cfaabe7}
```

Comment:
Per the Google style guide, use sentence case capitalization for headings. Please apply sentence case caps to all headings in this PR.